core/domain/exp_domain_test.py

```diff
@@ -4000,8 +4114,8 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
       customization_args:
         choices:
           value:
-          - <p>This is </p><p>value1 <br>for MultipleChoice</p>
-          - <p>This is value2 for <br>MultipleChoice</p>
+          - <p>This is </p><p>value1 <br/>for MultipleChoice</p>
```

Comment:
No, it should be `<br>`. /cc @bansalnitish

Reply:
Hi @ischur7, it should be <br>. See my comment here: https://github.com/oppia/oppia/blob/develop/core/domain/html_cleaner.py#L468

Reply:
Hi @ischur7, it should be <br>. See my comment here: https://github.com/oppia/oppia/blob/develop/core/domain/html_validation_service.py#L337

Reply:
See this [link](https://github.com/oppia/oppia/pull/5335#discussion_r204937850).
.
You have written in the comment that it sometimes automatically changes th <br> to <br/>
Maybe that is what is happening here. Because the tests fail if I keep it <br>. And I haven't altered anything related to this manipulation.
Thanks

Reply:
Can you work with @bansalnitish on this?

If tests pass in develop, and tests fail in this branch, then something is wrong with this branch. Don't change the test.

Reply:
Yeah, discussed with Nitish and rermoved.